src/views/TechnicalAssistance/RequestForm/Basic.tsx

```diff
@@ -1,34 +1,685 @@
+                <Label htmlFor="name" error={!!error}>
+                  {t('basic.labels.name')}
+                </Label>
+                {error && (
+                  <ErrorMessage>{t('basic.errors.fillBlank')}</ErrorMessage>
+                )}
+                <TextInput
+                  {...field}
+                  ref={null}
+                  id="name"
+                  type="text"
+                  validationStatus={error && 'error'}
+                />
+              </FormGroup>
+            )}
+          />
+
+          {/* Request component */}
+          <Controller
+            name="component"
+            control={control}
+            render={({ field, fieldState: { error } }) => (
+              <FormGroup error={!!error}>
+                <Label
+                  htmlFor="component"
+                  hint={<div>{t('basic.hint.component')}</div>}
+                  error={!!error}
+                >
+                  {t('basic.labels.component')}
+                </Label>
+                {error && (
+                  <ErrorMessage>{t('basic.errors.makeSelection')}</ErrorMessage>
+                )}
+                <Dropdown
+                  id="component"
+                  data-testid="component"
+                  {...field}
+                  ref={null}
+                >
+                  <option>- {t('basic.options.select')} -</option>
+                  {cmsDivisionsAndOfficesOptions('component')}
+                </Dropdown>
+              </FormGroup>
+            )}
+          />
+
+          {/* What do you need technical assistance with? */}
+          <Controller
+            name="needsAssistanceWith"
+            control={control}
+            render={({ field, fieldState: { error } }) => (
+              <FormGroup error={!!error}>
+                <Label
+                  htmlFor="needsAssistanceWith"
+                  hint={<div>{t('basic.hint.needsAssistanceWith')}</div>}
+                  error={!!error}
+                >
+                  {t('basic.labels.needsAssistanceWith')}
+                </Label>
+                {error && (
+                  <ErrorMessage>
+                    {t('basic.errors.includeExplanation')}
+                  </ErrorMessage>
+                )}
+                <CharacterCount
+                  {...field}
+                  ref={null}
+                  id="needsAssistanceWith"
+                  maxLength={2000}
+                  isTextArea
+                  rows={2}
+                  aria-describedby="needsAssistanceWith-info needsAssistanceWith-hint"
+                  error={!!error}
+                />
+              </FormGroup>
+            )}
+          />
+
+          {/* Do you have a solution in mind already? */}
+          <Controller
+            name="hasSolutionInMind"
+            control={control}
+            render={({ field, fieldState: { error }, formState }) => {
```

Comment:
There are a lot of these TS warning on these controller component's render params.  I wonder if there is any way to define these further up in scope to apply to these params

Reply:
I do not get ts warnings for these repeating params if that's what you're referring to. The client typecheck is fine. I've used the no-shadow exception because of the nesting of controllers. It's not common to need a reference to the parent param of the same name, but when it's necessary then they are named distinctly, like in the case of `startOrEndError`.